src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -70,12 +73,28 @@ public function doRun(InputInterface $input, OutputInterface $output)
 
         $this->setDispatcher($this->kernel->getContainer()->get('event_dispatcher'));
 
+        if ($this->registrationErrors) {
+            $this->renderRegistrationErrors($output);
```

Comment:
not sure to understand. Lazy command names are known beforehand and cannot fail before being ran since they are lazy, nothing to change for them to me